student.rb

```diff
@@ -1,8 +1,8 @@
 require '.person'
 
 class Student < Person
-  def initialize(classroom)
-    super(name, age, parent_permission: true)
+  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] [OPTIONAL] Kindly pass the parent_permission with colon notation as the parent_permission already use this colon notation in the parent class so it will be like:
```
super(age, name, parent_permission: parent_permission)
```